cmd/sonobuoy/app/sonobuoyconfig.go

```diff
@@ -56,12 +61,7 @@ func (c *SonobuoyConfig) Set(str string) error {
 	return nil
 }
 
-// Get will return the config.Config if one is available, otherwise nil.
+// Get will return the config.Config.
 func (c *SonobuoyConfig) Get() *config.Config {
-	// Don't just return zero structs
```

Comment:
Just initialize the method to the default which is normally what you do with flags. Not sure why we did it this way at first.